examples/proxy/envelope-encryption/PREPARE_KMS.md

```diff
@@ -0,0 +1,127 @@
+# Preparing a Key Management Service (KMS)
+
+As discussed [here](./README.md) you must provide a KMS solution in order to use the Envelope Encryption Filter.
+For Tech Preview, only HashiCorp Vault is supported.
+
+## Preparing HashiCorp Vault
+
+You may either use an existing instance of HashiCorp Vault you have available in your organisation, or deploy a development instance
```

Comment:
```suggestion
You may either use an existing instance of HashiCorp Vault you have available in your organisation, or deploy a development instance.
```

Reply:
Can't we use a production instance?

Reply:
Technically speaking, any Vault instance with the transit engine enable will function just fine.  Should the customer be using their production Vault instance to experiment with a Tech Preview feature? Probably no.   Do we need to spell that out?